### PR TITLE
Added Convolutional Neural Network

### PR DESCRIPTION
**Pull request by Shreyashmohadikar**
# Description

> ### :memo: Convolutional Neural Network

## Type of change

- [x] New layer/algorithm

numpy==1.23.0
matplotlib==3.5.2
pandas==1.4.3
tensorflow==2.9.1
> * Test score of 1 instance :
 loss: 0.4993, Accuracy: 0.8444,  val_loss: 0.5890, val_accuracy: 0.7667

## Checklist

- [x] I have performed a self-code_review of my own code.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by HemanthSai7 — COMMENTED** [review]
LGTM!🔥

**Review by mayureshagashe2105 — APPROVED** [review]
Nice work! Looks good to me! Merging!
